src/main/java/codeu/model/data/Event.java

```diff
@@ -84,16 +90,66 @@ public Event(Message message) {
     titleOfConversation = null;
     authorIdForConversation = null;
     conversationCreationTime = null;
+    conversationId = null;
 
     messageCreationTime = message.getCreationTime();
     authorIdForMessage = message.getAuthorId();
-    conversationTitleOfMessage = conversationStore.getConversationWithId(message.getConversationId()).getTitle(); //null error is here
+    try {
+      conversationStore.getConversationWithId(message
+        .getConversationId()).getTitle();
+    }
+    catch (NullPointerException e) {
+      System.err.println("_______________________________________________"  +
+                         "\n CAUGHT EXCEPTION_____________________________" + 
```

Comment:
nit:
You can keep this. But generally, unit test is to serve the app code. Normally do NOT put any logic/message specific for unit test in the application code.
For example, let say the app is running for some reason this NullPointerException is trigger, then the user will see a message about unit test, that is weird.

Reply:
Haha got it.